test/math/CMakeLists.txt

```diff
@@ -17,6 +17,10 @@ add_unit_test(SparseMatrixCSRVector2General
         math/SparseMatrixCSR.cpp
     )
 
+add_unit_test(Polynomial
+        math/Polynomial.cpp
```

Comment:
This should be redundant. The `add_unit_test` macro should automatically add the corresponding cpp file.